lua/lspconfig/julials.lua

```diff
@@ -51,13 +57,20 @@ configs.julials = {
     description = [[
 https://github.com/julia-vscode/julia-vscode
 
-`LanguageServer.jl` can be installed with `julia` and `Pkg`:
+LanguageServer.jl can be installed with `julia` and `Pkg`:
 ```sh
-julia -e 'using Pkg; Pkg.add("LanguageServer"); Pkg.add("SymbolServer")'
+julia --project=~/.julia/environments/nvim-lspconfig -e 'using Pkg; Pkg.add("LanguageServer")'
```

Comment:
In Julia 1.7 and above this can be spelled
```
julia --project=@nvim-lspconfig -e 'using Pkg; Pkg.add("LanguageServer")'
```
but lets wait (at least) until that is released before recommending that.